app/views/request/_after_actions.html.erb

```diff
@@ -5,7 +5,7 @@
     <strong><%= _('Anyone:') %></strong>
 
     <ul>
-      <% if @info_request.comments_allowed? %>
+      <% if AlaveteliConfiguration.enable_annotations && @info_request.comments_allowed? %>
```

Comment:
I don't have a strong preference, but I would find that more confusing I think - it's a boolean field on the model, so the database and the ruby object would be telling different stories about what's happening. It also seems like giving the model too much extra knowledge about the application state?